### PR TITLE
Remove unused stuff

### PR DESCRIPTION
**Pull request by mathias-lang-sociomantic**
Just minor things that were obvious while looking at the code.

### CODE REVIEW

**Comment by yebblies** [comment]
> 	Remove unused and hidden debug switches

These are not unused.

**Comment by Geod24** [comment]
> These are not unused.

Looks like grep failed me this time. Removed.

File: src/mars.d [hunk 1]
```diff
@@ -624,7 +619,7 @@ Language changes listed by -transition=id:
   =import,10378  revert to single phase name lookup
   =tls           list all variables going into thread local storage
 ");
-                        return EXIT_FAILURE;
+                        exit(EXIT_SUCCESS);
```
Comment (yebblies) [inline src/mars.d]: Doesn't using exit instead of return break some druntime stuff?

Comment (Geod24) [inline src/mars.d]: Then it should already be broken, as `-man`, `-version` and `-help` are using the same approach.

Comment (yebblies) [inline src/mars.d]: Fair enough.

**Comment by yebblies** [comment]
Auto-merge toggled on

**Pull request merged**